packages/@o3r/application/package.json

```diff
@@ -15,16 +15,18 @@
     "prepare:publish": "dispatch-package-json-fields ./dist/package.json"
   },
   "peerDependencies": {
+    "@angular-devkit/schematics": "~15.0.4",
     "@angular/common": "^15.0.2",
     "@angular/core": "^15.0.2",
     "@o3r/core": "workspace:^",
-    "@o3r/dev-tools": "workspace:^",
     "@o3r/logger": "workspace:^",
     "@o3r/routing": "workspace:^",
     "@o3r/testing": "workspace:^",
     "rxjs": "^7.4.0"
   },
   "dependencies": {
+    "@o3r/dev-tools": "workspace:^",
+    "@o3r/schematics": "workspace:^",
```

Comment:
if `@o3r/schematics` is added in the project devDep by `ng add @o3r/core` it could remain a peerDeps and be removed to  both `ng-package.json` and dep of the `package.json`

Reply:
indeed. updated